src/atomate2/common/schemas/magnetism.py

```diff
@@ -0,0 +1,207 @@
+    energy: float = Field(None, description="Final energy result from the calculation.")
+    energy_per_atom: float = Field(None, description="Final energy per atom.")
+
+    @classmethod
+    def from_task_document(
+        cls, task_document, uuid: str | None = None
+    ) -> MagneticOrderingOutput:
+        """
+        Construct a MagneticOrderingRelaxation output doc from a task document. This is
+        to be implemented for the DFT code of choice.
+        """
+        raise NotImplementedError
+
+
+class MagneticOrderingOutput(BaseModel):
+    """
+    Defines the output for a *static* magnetic ordering calculation. This is used
+    within the construction of the MagneticOrderingDocument. If a relaxation was
+    performed, this information will be stored within the relax_output field.
+    """
+
+    uuid: str = Field(None, description="Unique ID of the calculation.")
+    dir_name: str = Field(None, description="Directory of the calculation.")
+    input: MagneticOrderingInput = Field(
+        None, description="Input ordering information."
+    )
+    structure: Structure = Field(
+        None, description="Final structure from the calculation."
+    )
+    ordering: Ordering = Field(
+        None,
+        description=(
+            "The magnetic ordering of the output structure, "
+            "as defined in pymatgen.analysis.magnetism.analyzer."
+        ),
+    )
+    magmoms: list[float] = Field(None, description="Magnetic moments of the structure.")
+    symmetry: str = Field(None, description="Detected space group symbol.")
+    energy: float = Field(None, description="Final energy result from the calculation.")
+    energy_per_atom: float = Field(None, description="Final energy per atom.")
+    total_magnetization: float = Field(
+        None,
+        description=(
+            "Total magnetization as a sum of individual atomic moments in "
+            "the calculated unit cell."
+        ),
+    )
+    total_magnetization_per_formula_unit: float = Field(
+        None, description="Total magnetization normalized to per formula unit."
+    )
+    total_magnetization_per_unit_volume: float = Field(
+        None, description="Total magnetiation noramlized to per unit volume."
+    )
+    ordering_changed: bool = Field(
+        None,
+        description=(
+            "Specifies whether or not the magnetic ordering changed during the"
+            " calculation."
+        ),
+    )
+    symmetry_changed: bool = Field(
+        None,
+        description=(
+            "Specifies whether or not the symmetry changed during the calculation."
+        ),
+    )
+    energy_above_ground_state_per_atom: float = Field(
+        None, description="Energy per atom above the calculated ground state ordering."
+    )
+    relax_output: MagneticOrderingRelaxation | None = Field(
+        None, description="Relaxation output, if relaxation performed."
+    )
+    energy_diff_relax_static: str | float | None = Field(
```

Comment:
Might be worth noting this field was only really present for the purpose of benchmarking (i.e., is the static calculation required).

Since I wrote the original workflow, I have since concluded that the static step should--at minimum--have a much smaller EDIFF, however I have not benchmarked this. The original benchmark results speak for themselves, however the difference in energies between orderings is sometimes very very small, such that I would have more confidence if we had a tighter tolerance.

Reply:
This is an important (and still unresolved point). Right now, the default static calculations in the workflow are made with the VASP `StaticMaker`, which I believe is `EDIFF=1e-5`.

Reply:
The default `StaticMaker` (EDIFF=1e-5) is looser than `TightRelaxMaker` (ENCUT=700, EDIFFG=1e-7). Maybe we could use that parameter and this `energy_diff_relax_static` term to benchmark and see if we need to improve the EDIFF in `StaticMaker`.